tests/client_test.py

```diff
@@ -156,6 +156,7 @@ def test_client_get_server_commands(self):
         for cmd in ["connect", "info", "str"]:
             assert cmd in cmds
 
+    @pytest.mark.skip_if_max_rank_greater_than(9)
```

Comment:
woah, did someone actually build with `>=10` dims?? lol

Reply:
Someday they might?....?

Reply:
oh yah it's def a good addition! I was just shocked u found it :)

Reply:
You can congratulate Andy, it was his idea :)

Reply:
@drculhane nice find!! 🎉